docs/book/src/user/quick-start.md

```diff
@@ -310,6 +310,21 @@ You can now create your first workload cluster by running the following:
   clusterctl config cluster [name] --kubernetes-version [version] | kubectl apply -f -
 ```
 
+Optionally, you may want to check if the components are running properly. The
+exactly components are depend on which providers you have initialized, below is an example output
+with docker provider being installed.
+
+```
+kubectl get deploy -A | grep  "cap\|cert"
+capd-system                         capd-controller-manager                         1/1     1            1           25m
+capi-kubeadm-bootstrap-system       capi-kubeadm-bootstrap-controller-manager       1/1     1            1           25m
+capi-kubeadm-control-plane-system   capi-kubeadm-control-plane-controller-manager   1/1     1            1           25m
+capi-system                         capi-controller-manager                         1/1     1            1           25m
+cert-manager                        cert-manager                                    1/1     1            1           27m
+cert-manager                        cert-manager-cainjector                         1/1     1            1           27m
+cert-manager                        cert-manager-webhook                            1/1     1            1           27m
+```
+
```

Comment:
Let's keep this PR scoped to the developer doc.
My suggestion is to expand the "Create a kind management cluster" paragraph by adding something like

```
When the kind cluster is ready and all the required images are in place, run the 
clusterctl init command generated by the create-local-repository.py script. 

Optionally you might want to check if the components are running properly (this paragraph)
```

Reply:
done.